src/Month.java

```diff
@@ -0,0 +1,46 @@
+public class Month {
+    static void monthNumber(int number) {
+
+        switch (number) {
+            case 1:
+                System.out.println("January - 31 days");
+                break;
+            case 2:
+                System.out.println("February - 28 days");
+                break;
+            case 3:
+                System.out.println("March - 31 days");
+                break;
+            case 4:
+                System.out.println("April - 30 days");
+                break;
+            case 5:
+                System.out.println("May - 28 days");
+                break;
+            case 6:
+                System.out.println("June - 30 days");
+                break;
+            case 7:
+                System.out.println("July - 31 days");
+                break;
+            case 8:
+                System.out.println("August - 31 days");
+                break;
+            case 9:
+                System.out.println("September - 30 days");
+                break;
+            case 10:
+                System.out.println("October - 31 days");
+                break;
+            case 11:
+                System.out.println("November - 30 days");
+                break;
+            case 12:
+                System.out.println("December - 31 days");
+                break;
+        }
```

Comment:
default case should be added

Reply:
Implementation avoiding duplication
 static void monthDaysCounter(int monthNumber) {
        switch (monthNumber) {
            case 1:
            case 3:
            case 7:
            case 8:
            case 10:
            case 12:
                System.out.println(monthNumber + " month of the 2023 Year contains 31 days");
                break;
            case 4:
            case 6:
            case 9:
            case 11:
                System.out.println(monthNumber + " month of the 2023 Year contains 30 days");
                break;
            case 2:
                System.out.println(monthNumber + " month of the 2023 Year contains 28 days");
                break;
            default:
                System.out.println("Sorry, you are not right, we have only 12 months in a year");
        }
    }